src/tsickle.ts

```diff
@@ -903,12 +933,6 @@ class ExternsWriter extends ClosureRewriter {
           break;
         case ts.SyntaxKind.MethodSignature:
         case ts.SyntaxKind.MethodDeclaration:
-          let m = <ts.MethodDeclaration>member;
-          this.emitFunctionType(m);
-          this.emit(
-              `${typeName}.prototype.${m.name.getText()} = ` +
```

Comment:
Oh, I see the getText() was my own fault.  Feel free to ignore my other comment. :P